packages/launcher/lib/linux/index.ts

```diff
@@ -36,22 +44,30 @@ function getLinuxBrowser (
     throw notInstalledErr(binary)
   }
 
+  const maybeSetSnapProfilePath = (versionString: string) => {
+    if (os.platform() === 'linux' && name === 'chromium' && versionString.endsWith('snap')) {
+      // when running as a snap, chromium can only write to one directory
+      // @see https://cs.chromium.org/chromium/src/chrome/installer/linux/snap/snapcraft.yaml.in?l=59-61&rcl=6fb05bb3738f37392dde9cd2783637f09d5aa3f4
```

Comment:
the dir this yaml file indicates actually isn't writable from inside the sandbox - but this `~/snap/chromium/current` symlink is